### ISSUE TITLE
Swift 3.2 (Xcode 9 Beta 5) Expression was too complex to be solved in reasonable time

### ISSUE THREAD

**Issue by akac** [SwiftyBeaver/SwiftyBeaver#242]
AES256CBC.swift:495:23: Expression was too complex to be solved in reasonable time; consider breaking up the expression into distinct sub-expressions

**Comment by lgaches** [comment]
It fixed in #241

**Comment by skreutzberger** [comment]
Actually a user just sent a support message with the following content. It is obselete with @lgaches PR?

> I installed Xcode 9 beta 5 last night and it broke SwiftyBeaver 1.3.2.
> 
> To my surprise I was able to fix it by altering the code below as shown. The complaint in Xcode was that the original 4 expressions for rk2 were too complex to "complete in a reasonable amount of time" or whatever the wording is. My fix did the job.
> 

```swift
for r in 1..<rounds {
var w: UInt32
w = rk2[r][0]
var x1 = Int(B0(w))
var x2 = Int(B1(w))
var x3 = Int(B2(w))
var x4 = Int(B3(w))

rk2[r][0] = U1[x1] ^ U2[x2] ^ U3[x3] ^ U4[x4]

w = rk2[r][1]
x1 = Int(B0(w))
x2 = Int(B1(w))
x3 = Int(B2(w))
x4 = Int(B3(w))
rk2[r][1] = U1[x1] ^ U2[x2] ^ U3[x3] ^ U4[x4]

w = rk2[r][2]
x1 = Int(B0(w))
x2 = Int(B1(w))
x3 = Int(B2(w))
x4 = Int(B3(w))
rk2[r][2] = U1[x1] ^ U2[x2] ^ U3[x3] ^ U4[x4]

w = rk2[r][3]
x1 = Int(B0(w))
x2 = Int(B1(w))
x3 = Int(B2(w))
x4 = Int(B3(w))
rk2[r][3] = U1[x1] ^ U2[x2] ^ U3[x3] ^ U4[x4]
}
```

**Comment by skreutzberger** [comment]
Laurent's PR which brought official support for Swift 3.2 and Swift 4 was released today as SwiftyBeaver 1.4.0. An update should resolve the issue.
